pkg/injector/webhook.go

```diff
@@ -204,7 +211,9 @@ func (wh *webhook) mutate(req *v1beta1.AdmissionRequest) *v1beta1.AdmissionRespo
 
 	// Create the patches for the spec
 	// We use req.Namespace because pod.Namespace is "" at this point
-	patchBytes, err := wh.createPatch(&pod, req.Namespace)
+	// This string uniquely identifies the pod. Ideally this would be the pod.UID, but this is not available at this point.
+	proxyUUID := uuid.New().String()
+	patchBytes, err := wh.createPatch(&pod, req.Namespace, proxyUUID)
```

Comment:
I'd prefer to revert this change since it seems unnecessary.

Reply:
@shashankram I noticed that `createPatch()` was creating the proxyUUID.  I think that `createPatch()` should not know what proxy UUID is or how it is generated. So I decided to pull it out of `createPatch()`.
This makes testing `createPatch()` a bit easier too.